onadata/libs/serializers/organization_member_serializer.py

```diff
@@ -108,12 +108,18 @@ def create(self, validated_data):
             user = User.objects.get(username=username)
 
             if remove:
-                remove_org_user_async.apply_async(args=[organization.pk, user.pk])
+                remove_user_from_organization(organization, user)
 
             else:
-                add_org_user_and_share_projects_async.apply_async(
-                    args=[organization.pk, user.pk, role, email_subject, email_msg]
-                )
+                add_user_to_organization(organization, user, role)
+
+                if email_msg and email_subject and user.email:
+                    send_mail(
+                        email_subject,
+                        email_msg,
+                        settings.DEFAULT_FROM_EMAIL,
+                        (user.email,),
+                    )
```

Comment:
Is send mail, sync or async?

Reply:
@FrankApiyo It's sync, ideally it should be async, but I was indifferent in making the change at the moment. We should have `send_email_async` task should be available for the entire project.